### PR TITLE
Packaging infotheo version 0.0.5

### PR DESCRIPTION
**Pull request by affeldt-aist**
Upgrade to Coq 8.10.

### CODE REVIEW

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "reynald.affeldt@aist.go.jp"
+homepage: "https://github.com/affeldt-aist/infotheo"
+bug-reports: "https://github.com/affeldt-aist/infotheo/issues"
+dev-repo: "git+https://github.com/affeldt-aist/infotheo.git"
+license: "GPLv3"
```
Comment (palmskog) [inline released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]:
```suggestion
license: "GPL-3.0-only"
```
We are trying to switch all projects to [SPDX identifiers](https://spdx.org/licenses/). Unfortunately, "GPLv3" is ambiguous. Can you confirm that this is _only_ GPL 3 and not GPL 3 _or later_? The upstream repo doesn't seem to offer clarity on this.

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "Taku Asai"
+  "Takafumi Saikawa"
+  "Naruomi Obata"
+  "Erik Martin-Dorel"
+  "Ryosuke Obi"
+  "Mitsuharu Yamamoto"
+]
+build: [
+  ["coq_makefile" "-f" "_CoqProject" "-o" "Makefile"]
+  [make]
```
Comment (palmskog) [inline released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]:
```suggestion
  [make "-j%{jobs}%"]
```
Is there any specific reason to disable parallelism here? 

I would also advice that the `coq_makefile` command is moved to a custom Makefile inside the project, but that would be for future releases.

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "category:Computer Science/Data Types and Data Structures"
+  "keyword: information theory"
+  "keyword: probability"
+  "keyword: error-correcting codes"
+  "logpath:infotheo"
+  "date:2019-11-09"
+]
+url {
+  http: "https://github.com/affeldt-aist/infotheo/archive/0.0.5.tar.gz"
+  checksum: "43b937f49aafaae9d537e61c77345f8f"
```
Comment (palmskog) [inline released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]:
```suggestion
  checksum: "sha512=0ba8876041b815a624b49721521d31334858c6ecce941b5ba8ba29e374bd9bb300be9dc197dba0478a0def20154c3363e46bfe795dfff736a20d31dd05a8ef9a"
```
We are trying to deprecate MD5 for checksums.

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam (hunk 1 above)
Comment (affeldt-aist) [reply released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]: changed to GPL 3 or later

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam (hunk 2 above)
Comment (affeldt-aist) [reply released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]: parallelism enabled (custom makefile deferred to later)

File: released/packages/coq-infotheo/coq-infotheo.0.0.5/opam (hunk 3 above)
Comment (affeldt-aist) [reply released/packages/coq-infotheo/coq-infotheo.0.0.5/opam]: done

**Comment by affeldt-aist** [comment]
Thank you for your comments. I updated the opam file.

**Comment by palmskog** [comment]
Thanks for the clarifications and updates, I will merge as soon as CI passes.

**Pull request merged**